thundra/wrappers/flask/flask_executor.py

```diff
@@ -0,0 +1,61 @@
+from thundra import constants
+from thundra.plugins.invocation import invocation_support
+
+from thundra.wrappers import wrapper_utils, web_wrapper_utils
+
+
+def start_trace(plugin_context, execution_context, tracer):
+    request = execution_context.platform_data['request']
+
+    _request = {
+        'method': request.method,
+        'host': request.host.split(':')[0],
+        'query_params': request.query_string,
+        'body': request.data,
+        'headers': request.headers,
+        'path': request.path
+    }
+
+    web_wrapper_utils.start_trace(execution_context, tracer, 'Flask', 'API', _request)
+
+    try:
+        if execution_context.scope:
```

Comment:
Is it possible to pass this request.url_rule property as additional and optional parameter to `web_wrapper_utils.start_trace`. So if this parameter is passed, it sets this parameter to 
- root span operation name
- trigger operation name
- application resource name
- invocation trigger operation name tag

So it won't be required to update these values here